### PR TITLE
[FR] Add `Guides\osu!mania_Mapping_Guide`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal for the French translation of the wiki page `osu!mania_Mapping_Guide`.

Also:

- Addition of the French images
- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

*Feel free to merge if the delay is too long for a review by a native.*

### CODE REVIEW

File: wiki/Guides/osu!mania_Mapping_Guide/fr.md [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+Le premier décalage est le premier temps de votre BPM actuel. Allez au point de votre chanson où vous pouvez entendre le premier temps, appuyez sur F6, cliquez sur le point de timing et appuyez sur "Use current time". Maintenant, le point de synchronisation devrait se trouver sur le décalage actuel, le premier temps.
+
+S'il est juste, votre chanson est enfin chronométrée !
+
+#### MixMeister BPM Analyzer
+
+Nous voudrions aussi vous montrer un petit programme qui peut être très utile pour trouver le bon BPM très rapidement. Il s'appelle ***MixMeister BPM Analyzer***. Ce programme vous montrera le BPM moyen de votre chanson. Il a juste un point faible. Il ne peut pas vous montrer plusieurs BPM. Il vous montrera le BPM moyen de tous les points de BPM de votre chanson. Pour le dire simplement : Il n'aide qu'avec un seul BPM. Vous pouvez le rechercher sur Google ou simplement cliquer [ici] (https://dropbox.com/s/m4pjenvo4n65943/bpmanalyzer.zip?dl=0). N'essayez pas de l'utiliser pour timer toutes vos beatmaps. Vous n'apprendrez jamais à timer des musiques si vous laissez ce programme timer tout pour vous. Utilisez-le pour le comparer avec votre BPM pour vérifier si vous avez le bon !
```
Comment (DoryanR) [inline wiki/Guides/osu!mania_Mapping_Guide/fr.md:135]:
```suggestion
Nous voudrions aussi vous montrer un petit programme qui peut être très utile pour trouver le bon BPM très rapidement. Il s'appelle ***MixMeister BPM Analyzer***. Ce programme vous montrera le BPM moyen de votre chanson. Il a juste un point faible. Il ne peut pas vous montrer plusieurs BPM. Il vous montrera le BPM moyen de tous les points de BPM de votre chanson. Pour le dire simplement : Il n'aide qu'avec un seul BPM. Vous pouvez le rechercher sur Google ou simplement cliquer [ici](https://dropbox.com/s/m4pjenvo4n65943/bpmanalyzer.zip?dl=0). N'essayez pas de l'utiliser pour timer toutes vos beatmaps. Vous n'apprendrez jamais à timer des musiques si vous laissez ce programme timer tout pour vous. Utilisez-le pour le comparer avec votre BPM pour vérifier si vous avez le bon !
```

File: wiki/Guides/osu!mania_Mapping_Guide/fr.md [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+
+Les détails sur les snaps à utiliser pour le classement sont détaillés dans les [critères de classement d'osu!mania](/wiki/Ranking_Criteria/osu!mania).
+
+*Raccourci :* `Ctrl + Molette de la souris`
+
+**8. Sample Set & Additions** : Les Sample Sets et les Additions vous permettent d'ajouter encore plus de hitsounds que ceux présentés au point 5. Par exemple, vous pouvez superposer un son de fin de batterie au son de fin normal. Ainsi, vous pouvez donner à votre morceau une variété encore plus grande de sons frappés !
+
+*Raccourci :* `Ctrl + (W, E, R), Shift + (W, E, R)`
+
+![Éditeur d'osu!mania](img/Mania_Editor-FR.jpg "Éditeur d'osu!mania")
```
Comment (Walavouchey) [inline wiki/Guides/osu!mania_Mapping_Guide/fr.md:45]:
```suggestion
![Éditeur d'osu!mania](img/Mania_Editor-FR.png "Éditeur d'osu!mania")
```

Comment (Walavouchey) [reply wiki/Guides/osu!mania_Mapping_Guide/fr.md:45]: can be converted to jpg

**Review by Satsukel — CHANGES_REQUESTED** [review]
I don't have a lot of time at the moment but I'll try to review all your current PR. I didn't even have time to finish this one yet (might finish it later today tho). I wont have time to review until next week so I hope they wont be merged until then.

File: wiki/Guides/osu!mania_Mapping_Guide/fr.md [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
...à utiliser de courts pattern jack ici. Ce que vous devez éviter, ce sont les chord trills. Les chord trills ont une densité de pattern très élevée, ce qui peut entraîner un star ranting très élevé. Vous pouvez bien sûr utiliser de courts chord triples, mais ils ne devraient pas avoir un impact important sur le star ranting. Le kiai doit être la partie la plus dense de votre beatmap si possible. Aussi, vous pouvez essayer de combiner plus de LN et de 1/4 de pattern de notes simples. Essayez de combiner différentes longueurs de pattern LN en même temps ! Cela peut donner à votre beatmap une sensation spéciale en utilisant plus d'un LN en même temps. Mais fais très attention. Assurez-vous que chaque LN suit un son ! Ne les ajoutez pas "au hasard" dans votre musique juste parce que vous pensez que cela pourrait bien jouer. En fonction du BPM, il est également possible d'ajouter de courts pattern stairs 1/6 ou 1/8. Mais utilisez-les avec précaution. Ne les utilisez que si votre musique en offre suffisamment. Les stairs 1/6 ne doit pas être plus long qu'un 1/2 temps et le 1/8 pas plus long qu'un 1/4 temps ! Plus le bpm de votre chanson est bas, plus ces pattern stairs peuvent être longs.
```
Comment (Satsukel) [inline wiki/Guides/osu!mania_Mapping_Guide/fr.md:255]:
```suggestion
**La difficulté "Hard"** : Dans la difficulté Hard, nous pouvons commencer à ajouter des motifs plus compliqués car le joueur doit avoir appris les bases de Easy et Normal. Nous pouvons commencer à ajouter de plus longues salves de 1/4 et des pattern stairs, nous pouvons également commencer à utiliser des chords de 3 notes pour les cymbales lourdes à la fin ou au début d'une partie. De même, en fonction du bpm et de la partie de votre beatmap, nous pouvons utiliser des jumpstreams courts mais assurez-vous qu'ils correspondent à la partie que vous mappez actuellement. Lorsque la musique devient plus trépidante et plus rapide, c'est le moment idéal pour ajouter un tel jumpstream. Vous pouvez également commencer à utiliser de courts pattern jack ici. Ce que vous devez éviter, ce sont les chord trills. Les chord trills ont une densité de pattern très élevée, ce qui peut entraîner un star ranting très élevé. Vous pouvez bien sûr utiliser de courts chord triples, mais ils ne devraient pas avoir un impact important sur le star ranting. Le kiai doit être la partie la plus dense de votre beatmap si possible. Aussi, vous pouvez essayer de combiner plus de LN et de patterns 1/4 de notes simples. Essayez de combiner différentes longueurs de patterns LN en même temps ! Cela peut donner à votre beatmap une sensation particulière en utilisant plus d'une LN en même temps. Mais fais très attention. Assurez-vous que chaque LN suit un son ! Ne les ajoutez pas "au hasard" dans votre musique juste parce que vous pensez que cela pourrait bien jouer. En fonction du BPM, il est également possible d'ajouter de courts patterns stairs 1/6 ou 1/8. Mais utilisez-les avec précaution. Ne les utilisez que si votre musique en offre suffisamment l'occasion. Les stairs 1/6 ne doivent pas être plus long qu'un 1/2 temps et le 1/8 pas plus long qu'un 1/4 temps ! Plus le bpm de votre chanson est bas, plus ces pattern stairs peuvent être longs.
```

File: wiki/Guides/osu!mania_Mapping_Guide/fr.md [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
...rd mashing. Vous pouvez maintenant utiliser les chords à 3 notes pour faire plus que simplement souligner la fin d'une certaine partie de votre chanson. Mais n'oubliez pas : Si vous envisagez d'ajouter une autre difficulté, par exemple une difficulté supplémentaire, vous ne devez pas aller jusqu'aux limites ! Vous devez laisser un peu d'espace pour créer une difficulté encore plus difficile. Je ne peux pas vous dire grand-chose car vous devez simplement créer une difficulté plus difficile que la difficile dans une bonne répartition. Si vous prévoyez d'ajouter une difficulté supplémentaire, vous pouvez aller jusqu'aux limites de la chanson. Vous pouvez utiliser des jumpstreams lourds avec des chordmashs lourds. Vous pouvez utiliser des chords de 3 notes dans des jumpstreams entre des chords de 2 notes. Vous devez essayer de repousser les limites de la musique. Mais prenez note de ce qui suit : Une ligne directrice dit que vous devez être capable de passer vos propres difficultés et nous sommes d'accord avec ce point. Si vous voulez vraiment créer une bonne difficulté insane/extreme jouable, vous devez savoir comment elles se jouent et pour cela vous devez être capable de les passer.
```
Comment (Satsukel) [inline wiki/Guides/osu!mania_Mapping_Guide/fr.md:257]: Ce passage donne un peu mal au crane à certains moments je te conseille de le relire si jamais tu vois des choses qui vont pas niveau répétition mais franchement je vois pas comment faire plus clair sans répeter parfois x)
```suggestion
**Les difficultés "Insane" et "Extreme"** : Maintenant, nous nous concentrons sur les difficultés Insane et Extreme. Encore une fois, nous augmentons la densité des patterns et commençons à en utiliser des encore plus compliqués ! Vous pouvez maintenant utiliser toutes sortes de patterns et essayer de les combiner. De plus, vous pouvez maintenant utiliser des LN 1/8 ainsi que des chord trills ! Les difficultés Insane et Extreme sont surtout destinées aux joueurs plus expérimentés, donc vous devriez essayer de créer une difficulté vraiment agréable à jouer et toujours plus excitante. Essayez de combiner les jumpstreams avec les trills et le heavy chord mashing. Vous pouvez maintenant utiliser les chords à 3 notes pour faire plus que simplement souligner la fin d'une certaine partie de votre chanson. Mais n'oubliez pas : Si vous envisagez d'ajouter une autre difficulté, par exemple une difficulté supplémentaire, vous ne devez pas aller jusqu'aux limites ! Vous devez laisser un peu d'espace pour créer une difficulté encore plus difficile. Je ne peux pas vous dire grand-chose car dans une bonne répartition de difficulté vous devez simplement créer une difficulté plus difficile que la difficile. Si vous prévoyez d'ajouter une difficulté supplémentaire, vous pouvez aller jusqu'aux limites de la chanson. Vous pouvez utiliser des jumpstreams lourds avec des chordmashs lourds. Vous pouvez utiliser des chords de 3 notes dans des jumpstreams entre des chords de 2 notes. Vous devez essayer de repousser les limites de la musique. Mais prenez note de ce qui suit : Une ligne directrice dit que vous devez être capable de passer vos propres difficultés et nous sommes d'accord avec ce point. Si vous voulez vraiment créer une bonne difficulté insane/extreme jouable, vous devez savoir comment elles se jouent et pour cela vous devez être capable de les passer.
```

File: wiki/Guides/osu!mania_Mapping_Guide/fr.md [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+
+### Hitsounds
+
+Vous avez placé vos premières notes et cela vous plaît ? Ou bien vous avez même terminé une difficulté ou un spread complet ? Excellent ! Mais il manque quelque chose, non ? Oui ! Les Hitsounds.
+
+Les Hitsounds sont importants dans tous les modes de jeu. Ils donnent un retour d'information lorsque vous appuyez sur une note, ce qui permet au joueur de savoir ce qu'il vient de frapper. De plus, le fait d'avoir différents hitsounds dans votre beatmap peut lui donner un sentiment spécial car la musique sonne différemment tout d'un coup, peut-être même mieux que sans hitsounds ! osu!mania propose deux façons différentes d'utiliser les hitsounds.
```
Comment (Satsukel) [inline wiki/Guides/osu!mania_Mapping_Guide/fr.md:263]:
```suggestion
Les Hitsounds sont importants dans tous les modes de jeu. Ils donnent un retour d'information lorsque vous appuyez sur une note, ce qui permet au joueur de savoir ce qu'il vient de frapper. De plus, le fait d'avoir différents hitsounds dans votre beatmap peut lui donner une ambiance spéciale car la musique sonne différemment tout d'un coup, peut-être même mieux que sans hitsounds ! osu!mania propose deux façons différentes d'utiliser les hitsounds.
```

File: wiki/Guides/osu!mania_Mapping_Guide/fr.md [hunk 6]
```diff
@@ -0,0 +1,375 @@
...
+
+Tout d'abord, il y a la méthode normale qui utilise les échantillons et les effets fournis par le jeu lui-même, comme le sifflement/la fin et le clap. L'autre façon est via le menu d'échantillons que vous pouvez ouvrir avec `Ctrl` + `Shift` + `I`. Ce menu d'échantillons est particulièrement important pour le keysounding, c'est pourquoi nous l'expliquerons plus tard !
+
+Pour l'instant, nous voulons nous concentrer sur la méthode normale du hitsounding.
+
+Tout d'abord, vous devez décider si vous voulez utiliser les sons par défaut que osu! vous fournit ou si vous voulez ajouter des sons personnalisés.
+
+Si vous voulez utiliser celle par défaut, vous n'avez pas vraiment besoin de changer grand-chose. Tout ce dont vous avez besoin est de décider quel jeu d'échantillons vous voulez utiliser par défaut. Allez dans le panneau de configuration du timing et changez votre point de timing pour le jeu d'échantillons que vous voulez utiliser. osu!mania utilise généralement "soft" par défaut car le hit-normal n'est pas si fort.
```
Comment (Satsukel) [inline wiki/Guides/osu!mania_Mapping_Guide/fr.md:271]:
```suggestion
Si vous voulez utiliser ceux par défaut, vous n'avez pas vraiment besoin de changer grand-chose. Tout ce dont vous avez besoin est de décider quel jeu d'échantillons vous voulez utiliser par défaut. Allez dans le panneau de configuration du timing et changez votre point de timing pour le jeu d'échantillons que vous voulez utiliser. osu!mania utilise généralement "soft" par défaut car le hit-normal n'est pas si fort.
```

File: wiki/Guides/osu!mania_Mapping_Guide/fr.md [hunk 7]
```diff
@@ -0,0 +1,375 @@
...
+
+Vous pouvez toujours changer le sample set pour une partie entière en sélectionnant toutes les notes et en la changeant pour le sample set désirée ou vous pouvez ajouter un point hérité (ligne verte) et changer toute la partie pour le sample set sélectionnée jusqu'au prochain point hérité. Il ne vous reste plus qu'à ajouter le son que vous souhaitez ! Vous pouvez choisir entre Finish, Whistle et Clap. Tous ces effets sonnent différemment lorsque vous changez de jeu d'échantillons. Essayez un peu et je suis sûr que vous trouverez l'effet parfait pour votre musique !
```
Comment (Satsukel) [inline wiki/Guides/osu!mania_Mapping_Guide/fr.md:273]:
```suggestion
Vous pouvez toujours changer le sample set pour une partie entière en sélectionnant toutes les notes et en la changeant pour le sample set désiré ou vous pouvez ajouter un point hérité (ligne verte) et changer toute la partie pour le sample set sélectionné jusqu'au prochain point hérité. Il ne vous reste plus qu'à ajouter le son que vous souhaitez ! Vous pouvez choisir entre Finish, Whistle et Clap. Tous ces effets sonnent différemment lorsque vous changez de jeu d'échantillons. Essayez un peu et je suis sûr que vous trouverez l'effet parfait pour votre musique !
```

File: wiki/Guides/osu!mania_Mapping_Guide/fr.md [hunk 8]
```diff
@@ -0,0 +1,375 @@
...
+- drum-hitclap
+
+Vous pouvez ajouter plus que cela ! Vous pouvez également nommer vos hitsounds ! Par exemple "soft-hitnormal2". Vous devez changer le jeu d'échantillons que vous voulez utiliser dans le panneau de configuration du timing.
+**Note également :** Le "xxx-hitnormal" est le hitsound qui est joué lorsque vous ne donnez pas à la note un hitsound spécifique. Par exemple, lorsque vous changez le "soft-hitnormal", toutes les notes de votre map qui ont le sample set soft actif auront ce hitsound ! Soyez prudent lorsque vous les modifiez.
+
+Une fois que vous avez ajouté vos hitsounds, vous devez les activer !
+
+Pour cela, nous ouvrons le Timing Setup Panel dans l'éditeur et marquons tous les points. Maintenant, nous allons dans "Audio" dans le Timing Setup Panel et changeons le Sample Set en "Custom 1". Et voilà !
+
+Tous vos hitsounds du sample set 1 sont maintenant activés ! Si vous utilisez un autre sample set, choisissez le troisième point et choisissez le sample set que vous avez également choisie pour vos hitsounds.
```
Comment (Satsukel) [inline wiki/Guides/osu!mania_Mapping_Guide/fr.md:303]:
```suggestion
Tous vos hitsounds du sample set 1 sont maintenant activés ! Si vous utilisez un autre sample set, choisissez le troisième point et choisissez le sample set que vous avez également choisi pour vos hitsounds.
```

File: wiki/Guides/osu!mania_Mapping_Guide/fr.md [hunk 9]
```diff
@@ -0,0 +1,375 @@
...
+
+Une fois que vous avez enfin vos hitsounds, n'oubliez pas d'activer "Lock Notes". De cette façon, aucune note ne pourra être déplacée lorsque vous ferez un mouvement sur votre beatmap. Le Hitsounding ne prend pas beaucoup de temps. Vous pouvez choisir une ou plusieurs notes en même temps, changer le jeu d'échantillons en haut à gauche de l'écran et ajouter l'effet sur le côté droit. Maintenant, toutes ces notes ont un hitsound !
```
Comment (Satsukel) [inline wiki/Guides/osu!mania_Mapping_Guide/fr.md:305]:
```suggestion
Une fois que vous avez enfin vos hitsounds, n'oubliez pas d'activer "Lock Notes". De cette façon, aucune note ne pourra être déplacée pendant que vous modifiez votre beatmap. Le Hitsounding ne prend pas beaucoup de temps. Vous pouvez choisir une ou plusieurs notes en même temps, changer le jeu d'échantillons en haut à gauche de l'écran et ajouter l'effet sur le côté droit. Maintenant, toutes ces notes ont un hitsound !
```

File: wiki/Guides/osu!mania_Mapping_Guide/fr.md [hunk 10]
```diff
@@ -0,0 +1,375 @@
...
+
+Bien sûr, vous ne devez pas faire correspondre l'ensemble de la musique à la hauteur, car cela peut entraîner des problèmes d'équilibre dans votre beatmap.
+
+Ce que vous pouvez toujours essayer, c'est de faire correspondre les parties calmes de votre beatmap à la hauteur, surtout lorsque vous utilisez principalement des notes simples dans cette partie. De même, lorsque votre musique comporte des changements de hauteur instrumentale audibles, vous devez essayer de les suivre.
+
+Bien sûr, vous n'êtes pas obligé de suivre le pitch dans votre beatmap, mais vous pouvez le considérer comme un "petit plus" qui peut rendre votre beatmap plus intéressante à jouer.
+
+### Slider Velocity (SV)
+
+Le Slider Velocity agit un peu différemment dans osu!mania ; elle modifie la vitesse à laquelle les notes défilent vers le bas. Plus le Slider Velocity est élevée, plus les notes descendent vite, et plus il est faible, plus les notes descendent lentement.
```
Comment (Satsukel) [inline wiki/Guides/osu!mania_Mapping_Guide/fr.md:347]:
```suggestion
Le Slider Velocity agit un peu différemment dans osu!mania ; il modifie la vitesse à laquelle les notes défilent vers le bas. Plus le Slider Velocity est élevé, plus les notes descendent vite, et plus il est faible, plus les notes descendent lentement.
```

File: wiki/Guides/osu!mania_Mapping_Guide/fr.md [hunk 11]
```diff
@@ -0,0 +1,375 @@
...
+
+Bien que chaque personne qui fait du mapping ait son propre style, il existe quelques caractéristiques de base sur lesquelles vous pouvez reconnaître les styles de mapping des autres jeux.
+
+#### Stepmania Style
+
+Stepmania possède l'une des plus grandes communautés pour les jeux VSRG. Par conséquent, elle possède également une grande communauté de mappeurs !
+
+Le style Stepmania consiste, outre le style personnel de chaque mappeur, en deux styles de mapping différents. Le style NB5/Yolomania 4 et le style DCP/LCP.
+
+Le style DCP/LCP met l'accent sur la pertinence musicale grâce à des techniques de mapping telles que la pertinence de la hauteur et la théorie du jack. (La théorie du jack consiste essentiellement à créer des jacks pour mettre en valeur un certain instrument, comme un crash de cymbale). Les chartes DCP/LCP sont plus appréciées par les autres chartes que par les joueurs car elles se concentrent principalement sur la pertinence musicale et pas tellement sur la jouabilité.
```
Comment (Satsukel) [inline wiki/Guides/osu!mania_Mapping_Guide/fr.md:371]:
```suggestion
Le style DCP/LCP met l'accent sur la pertinence musicale grâce à des techniques de mapping telles que la pertinence de la hauteur et la théorie du jack. (La théorie du jack consiste essentiellement à créer des jacks pour mettre en valeur un certain instrument, comme un crash de cymbale). Les beatmaps DCP/LCP sont plus appréciées par leurs mappeurs que par les joueurs car elles se concentrent principalement sur la pertinence musicale et pas tellement sur la jouabilité.
```

File: wiki/Guides/osu!mania_Mapping_Guide/fr.md [hunk 12]
```diff
@@ -0,0 +1,375 @@
...
+
+Le style yolomania s'appuie plus sur le paternalisme qu'autre chose. Il s'attache davantage à créer de bons tableaux jouables pour le joueur et la communauté. Bien qu'il utilise toujours le concept de pertinence de la hauteur, il n'est pas de la plus haute importance.
```
Comment (Satsukel) [inline wiki/Guides/osu!mania_Mapping_Guide/fr.md:373]:
```suggestion
Le style yolomania s'appuie plus sur le patterning qu'autre chose. Il s'attache davantage à créer de bonnes beatmaps jouables pour le joueur et la communauté. Bien qu'il utilise toujours le concept de pertinence de la hauteur, il n'est pas non plus très important.
```

**Review by Satsukel — CHANGES_REQUESTED** [review]
ça devrait être bon après ça

**Review by Satsukel — DISMISSED** [review]

File: wiki/Guides/osu!mania_Mapping_Guide/fr.md [hunk 13]
```diff
@@ -0,0 +1,375 @@
...
+
+Allez au point où le BPM change, appuyez sur F6 pour ouvrir le panneau Timing, ajoutez un autre point de timing en cliquant sur le "plus" vert et ajustez-le au changement de BPM tout en écoutant à nouveau le métronome. Vous pouvez également appuyer sur Ctrl + P pour définir un point de BPM à l'horodateur où vous vous trouvez actuellement. Faites cela pour chaque changement de BPM dans votre beatmap !
+
+#### Timing Setup Panel
+
+![Timing Setup](/wiki/Beatmap_Editor/Timing/img/TimingSetup.png "Timing Setup")
+
+| Nom | Description |
+| :-- | :-- |
+| Timing Point : | Le point de synchronisation ou "ligne rouge" est là pour synchroniser votre chanson. Sans lui, vous ne pouvez pas commencer le mapping. Reportez-vous aux étapes ci-dessus pour [**timer*](#timing) votre chanson. |
```
Comment (Walavouchey) [inline wiki/Guides/osu!mania_Mapping_Guide/fr.md:149]:
```suggestion
| Timing Point : | Le point de synchronisation ou "ligne rouge" est là pour synchroniser votre chanson. Sans lui, vous ne pouvez pas commencer le mapping. Reportez-vous aux étapes ci-dessus pour [*timer*](#timing) votre chanson. |
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**